bundler/lib/bundler/templates/newgem/ext/newgem/Cargo.toml.tt

```diff
@@ -12,4 +12,11 @@ publish = false
 crate-type = ["cdylib"]
 
 [dependencies]
-magnus = { version = "0.6.2" }
+magnus = { version = "0.6.3" }
+rb-sys = { version = "0.9", features = ["stable-api-compiled-fallback"] }
```

Comment:
With this, we support `ruby-head` out of the box. This allows users to run development/unstable rubies in production without friction